### ISSUE TITLE
aranym does not use stdin & stdout

### ISSUE THREAD

**Issue by paulwratt** [aranym/aranym#64]
aranym prints to console but does not use stdin & stdout

`aranym --version > ara-ver.txt` does not work as expected, `ara-ver.txt` is empty

also you cant do `aranym 2>&1`

ie you can see text output, you just can capture it

**Comment by th-otto** [comment]
>aranym --version > ara-ver.txt does not work as expected

Yes, thats a small quirk, because all those message are printed using debug functions and thus go to stderr. Is that real an issue?

>also you cant do aranym 2>&1

I don't understand. "2>&1" just redirects stderr to stdout, so of course message still go to the console.

>aranym --version > ara-ver.txt 2>&1

will work as expected.

**Comment by paulwratt** [comment]
yes its a problem if you are working in the console (not X Windows) or if your terminal emulator does not have large scrollback history (AFROS-LiveCD)

especially if what you are looking for is before MiNT boot process, you dont have a chance to see anything

That said if it is outputting to `stderr` how come its not obvious in the wiki/manual?

**Comment by th-otto** [comment]
>yes its a problem if you are working in the console

But you can always redirect both to your file. You should do that anyway, because any messages printed through the NF_STDERR interface will go to stderr, too.
